client/telemetry/src/lib.rs

```diff
@@ -123,6 +123,12 @@ impl TelemetryEndpoints {
 	}
 }
 
+impl TelemetryEndpoints {
+    pub fn is_empty(&self) -> bool {
+        self.0.is_empty()
+    }
+}
+
```

Comment:
```suggestion
impl TelemetryEndpoints {
	/// Return `true` if there are telemetry endpoints, `false` otherwise.
	pub fn is_empty(&self) -> bool {
		self.0.is_empty()
	}
}

```